packages/css/src/utils/css-property.tsx

```diff
@@ -126,3 +128,17 @@ export function addUnitIfNeeded(
 
   return String(value).trim();
 }
+
+/**
+ * Returns `true` if the CSS property is defined within a template element
+ * (the string pieces of the template literal)
+ * rather than in the expression.
+ * Eg color: ${({ isPrimary }) => (isPrimary ? 'green' : 'red')};
+ * @param node
+ */
+export const isCssPropertyInTemplateElement = (node: t.TemplateElement): boolean => {
+  const re = /(\s.[a-z.\-_]*?\:)/; // `property:`
```

Comment:
- [ ] try not to use regex

Reply:
This won't work if we have
```
:focus { ${({ isTrue }) => isTrue ? 'color: purple' : 'color: orange'}; }
```

since we want to manipulate the node only if the CSS property is within the quasi, for example when we have
```
:focus: color { ${({ isTrue }) => isTrue ? 'purple' : 'orange'}; }
```

Reply:
Now that we are also including pseudo classes/pseudo elements we can just check if `:` is included.
Sorry for the confusion, the regex was initially designed to address the first issue ( CSS variables are created instead of classes)